app/cdap/components/common/InlayDrawerWidget/InlayDrawerWidget.stories.tsx

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
remove 'common' folder its unnecessary

Reply:
moved the component out of the common folder
the common folder will be removed one by one whenever the code inside that folders needs to be updated
we will keep moving the component out of this folder and finally delete it completely

hope this is fine